test/unit/mdc-slider/foundation.test.js

```diff
@@ -600,6 +724,25 @@ test('#setMin updates "aria-valuemin" to the new minimum', () => {
   raf.restore();
 });
 
+test('#setMin re-place track markers if slider is discrete and displays markers', () => {
```

Comment:
Is redo placement equivalent to replace? May be I should totally change how I say it... Any suggestions?